composer.json

```diff
@@ -23,16 +23,34 @@
     "mezzio-hal"
   ],
   "require": {
-    "php": "~7.4.0 || ~8.0.0 || ~8.1.0",
-    "mezzio/mezzio-hal": "^2.4.0"
+    "php": "~8.1.0 || ~8.2.0",
+    "mezzio/mezzio-hal": "^2.4.0",
+    "doctrine/common": "^3.4"
   },
   "require-dev": {
-    "phpunit/phpunit": "^9.5.20",
-    "squizlabs/php_codesniffer": "^3.6.2"
+    "phpunit/phpunit": "^10.2",
+    "laminas/laminas-coding-standard": "^2.5",
+    "vimeo/psalm": "^5.13"
   },
   "autoload": {
     "psr-4": {
       "Dot\\DoctrineMetadata\\": "src"
     }
+  },
+  "scripts": {
+    "check": [
+      "@cs-check",
+      "@test"
+    ],
+    "cs-check": "phpcs",
+    "cs-fix": "phpcbf",
+    "test": "phpunit --colors=always",
+    "test-coverage": "phpunit --colors=always --coverage-clover clover.xml",
+    "static-analysis": "psalm --shepherd --stats"
+  },
+  "config": {
```

Comment:
Not a requirement, rather a nice-to-have, but for consistency across all our packages, let's add under the `config` section:
`"sort-packages": true,`